subprojects/ivy/build.gradle.kts

```diff
@@ -14,7 +14,10 @@ dependencies {
     implementation(project(":file-collections"))
     implementation(project(":resources"))
     implementation(project(":publish"))
-    implementation(project(":plugins")) // for base plugin to get archives conf
+    implementation(project(":plugins")) {
+        because("For JavaPlatformPlugin")
```

Comment:
It looks like this should be inverted and the java/java-platform plugins should be configuring the version mapping strategy?

Reply:
Inverting this allowed us to remove this dependency.